pry-doc.gemspec

```diff
@@ -23,7 +23,7 @@ DESCR
   s.required_ruby_version = '>= 2.0'
 
   s.add_dependency 'yard', "~> 0.9.11"
-  s.add_dependency 'pry', "~> 0.9"
+  s.add_dependency 'pry', ">= 0.11.3"
```

Comment:
constrain this a bit more

Reply:
Can we bump versions and do other chores in a separate PR? This one is already quite big and this change doesn't seem to be relevant to the current work (unless I missed something).